tests/sentry/web/frontend/test_integration_setup.py

```diff
@@ -17,7 +17,7 @@ def test_basic_flow(self):
         assert resp.status_code == 200
         self.assertDialogSuccess(resp)
 
-        integration = Integration.objects.get(provider=self.provider.id)
```

Comment:
Do we need any more tests aside from the ones refactored here?

Reply:
I have some identity linking acceptance tests that I need to finish up, I'll try and get those in here, but I'd like to get this merged in sooner than later, I'll try and get those in, but otherwise they'll come in as a separate PR I think.

Other than that more tests will show up as we solidify more of what integration setup looks like from a configuration standpoint